### PR TITLE
Improved Mobile Experience - Fixed #5

### LINKED ISSUE #5: Header appears over content on mobile
[Example](https://i.imgur.com/hprMqGB.png)

### PR DESCRIPTION
**Pull request by SimplyCodin**
Fixes   #5 
Made header smaller on Mobile
Added Media querys for Responsive Design

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -31,7 +31,6 @@ themeColor: purple # green, blue, orange, purple, grey
 fixedNav: 'true' # true or false
 
 permalink: /:year/:month/:title
-markdown: kramdown
```
Comment (spotlightishere) [inline _config.yml]: What purpose does this have to be removed? Jekyll uses Kramdown for markdown right now, but it could change in the future.

**Comment by spotlightishere** [comment]
Actually, I was not able to confirm this worked on smaller devices. Will keep fiddling with.

**Comment by spotlightishere** [comment]
I manually merged this in 8d9277a835726e2af1d67fe014e84336a185de7b. Thank you so much, and apologies that it took so long to get this merged.

File: _config.yml (hunk 1 above)
Comment (SimplyCodin) [reply _config.yml]: It was in there twice

**Pull request closed without merging**